src/components/forms/FormFieldsEditor.tsx

```diff
@@ -67,10 +70,13 @@ const FormFieldsEditor = ({ form }: FormFieldsEditorProps) => {
       return;
     }
     updateForm.mutate(
-      { fields: fields, resource_type: form.resource_type },
+      { title: formTitle, fields: fields, resource_type: form.resource_type || 'Form' },
```

Comment:
Legg heller til `'form'` i resource_type-typen

```suggestion
      { title: formTitle, fields: fields, resource_type: form.resource_type },
```